### PR TITLE
Glossary

### PR DESCRIPTION
**Pull request by jasonnIguazio**
Added a glossary section to the documentation folders. @Hedingber Please review and let me know if this works.

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<a id="top"></a>
+
+# Glossary of Terms
+
+
+| MLRun Terms              | Definition                                                                                                                                                                                    |
+| -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FeatureSet               | Group of features that are ingested together and stored in logical group                                                                                                                      |
+| FeatureVector            | Combination of multiple FeatureSets with ability to join, filter, etc.                                                                                                                        |
```
Comment (yaronha) [inline docs/glossary.md:9]: of multiple Features originating from different FeatureSets

File: docs/glossary.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+| MLRun Function           | Abstraction that allows for running Python code easily in local environment and on top of K8s                                                                                                 |
```
Comment (yaronha) [inline docs/glossary.md:10]: its not Py specific, use "Abstraction over the code, extra packages, runtime configuration and desired resources which allow execution in local environment and on various serverless engines on top of K8s"

File: docs/glossary.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+| MLRun Marketplace        | Collection of pre-built MLRun functions avilable for usage                                                                                                                                    |
+| MLRun Project            | Logical separation of resources - includes model, jobs, real-time functions, datasets, feature sets, feature vectors, etc.                                                                    |
```
Comment (yaronha) [inline docs/glossary.md:12]: a logical container for all the work on a particular activity/application which include functions, workflow, artifacts, secrets, and more. and can be assigned to a specific group of users

File: docs/glossary.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+| **Iguazio (V3IO) Terms** | **Definition**                                                                                                                                                                                |
+| Consumer group           | Set of consumers which cooperate to consume data from some topics                                                                                                                             |
+| Key Value (KV) store     | Type of storage where data is stored by a specific key - allows for real-time lookups                                                                                                         |
+| TSDB                     | Time series database - part of V3IO                                                                                                                                                           |
+| V3IO                     | Iguazio data layer - supports several formats including KV, Block, File, Streams, and more                                                                                                    |
```
Comment (yaronha) [inline docs/glossary.md:21]: Iguazio **real-time** data layer

File: docs/glossary.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+| Dask                     | Flexible library for parallel computing in Python. Often used for data engieering, data science, and machine learning                                                                         |
+| KubeFlow pipeline        | Platform for building and deploying portable, scalable machine learning (ML) workflows based on Docker containers                                                                             |
+| Sklearn                  | Open source machine learning Python library. Used for modelling, pipelines, data transformations, feature engineering, and more.                                                              |
+| Spark                    | Open source parallel processing framework for running large-scale data analytics applications across clustered computers. Often used for data engineering, data science, and machine learning |
+| XGBoost                  | Optimized distributed gradient boosting library designed to be highly efficient, flexible and portable. Implements machine learning algorithms under the Gradient Boosting framework          |
+
```
Comment (yaronha) [inline docs/glossary.md:40]: i assume we need to add TensorFlow, Keras, PyTorch, TensorBoard (u can consult w @guy1992l  on it)

**Comment by george0st** [comment]
I am voting for adding these terms also:

- HttpDB
- Artifact
- HUB
- MPI

**Pull request closed without merging**